src/tools/dev/scripts/bv_support/bv_visit.sh

```diff
@@ -467,4 +467,9 @@ function bv_visit_build
             info "Finished with Errors"
         fi
     fi
+
+    if [[ $VISIT_BUILD_MODE == "Debug" ]]; then
+        info "Debug build mode was specified. The default build mode for VisIt is Release."
```

Comment:
This seems confusing to me. Perhaps, you can change it to:

Debug build mode was specified. This builds all the third party libraries and VisIt with debug support. To build just VisIt with debug support, build the third party libraries without VisIt and then build VisIt separately, passing -DCMAKE_BUILD_TYPE:STRING=Debug to VisIt's cmake configure command.

Reply:
Your suggestion makes sense as a general message about the build-mode flag. But not necessarily in this location.  This info message occurs after all the TP libs AND VisIt have been built. It is meant as an information message about how to build VisIt in debug mode.